### DISCUSSION TITLE
Is there any way to filter selected data with properties that are not selected?

### DISCUSSION THREAD

**Discussion by JulienLecoq** [PostgREST/postgrest#2528] [Q&A] [answered]
I have several places in my app where I select some data from a table and I would like to filter the selected data with data that are not selected. These data are part either of the selected table or part of some other table linked to my selected table thanks to a foreign key.

I can't find anything on that topic on the doc. That seems to be an important thing to be able to do, I don't want to waste bandwidth, computation time, etc, on things that I don't want to get on my app.

**Comment by steve-chavez** [comment]
Could you provide an example of what you want to achieve?

https://github.com/PostgREST/postgrest/issues/2340 might be related

**Comment by wolfgangwalther** [comment, marked as answer]
> These data are part either of the selected table

Regular horizontal filtering will work here: https://postgrest.org/en/stable/api.html#horizontal-filtering-rows

> part of some other table linked to my selected table thanks to a foreign key.

Not implemented, yet. Discussed here: #1414
